src/script.js

```diff
@@ -0,0 +1,105 @@
+let toDolist = [];
+
+function submit() {
+  let task = document.getElementById("task");
+  if (task.value != "") {
+    toDolist.push(task.value);
+    addToList(task.value, toDolist.length - 1);
+    task.value = null;
+    updateLocal();
+  }
+}
+
+function addToList(task, value) {
+  let desc = document.createTextNode(task);
+  let ol = document.getElementById("tasksList");
+  let li = document.createElement("li");
+  let rowContainer = document.createElement("div");
+  let deleteTask = document.createElement("button");
+  let textContainer = document.createElement("div");
+  deleteTask.setAttribute("onclick", "deleteTask(this.parentElement)");
+  deleteTask.setAttribute("content", "delete");
+  let isActive = document.createElement("input");
+  isActive.setAttribute("onclick", "isDone(this)");
+  isActive.setAttribute("type", "checkbox");
+  deleteTask.innerHTML = "x";
+  textContainer.appendChild(desc);
+  rowContainer.appendChild(isActive);
+  rowContainer.appendChild(textContainer);
+  li.appendChild(deleteTask);
+  li.appendChild(rowContainer);
+  rowContainer.setAttribute("class","rowContainer");
+  textContainer.setAttribute("class", "textContainer");
+  li.setAttribute("value", `${value}`);
+  ol.appendChild(li);
+}
+
+function updateLocal() {
+  if (toDolist.length > 0) {
```

Comment:
refactor this to first check if todoList is empty remove localstorage.removeitem, then do other functionality.